.env.example

```diff
@@ -13,7 +13,7 @@ TOTALVALIDATORS     = 10
 NUMVALIDATORS       = 4
 LOGCONFIG          = 'dev'  # dev/prod
 FLASK_LOG_LEVEL    = 'ERROR'  # DEBUG/INFO/WARNING/ERROR/CRITICAL
-DISABLE_INFO_LOGS_ENDPOINTS = '["get_transaction_by_hash","get_contract_schema_for_code","get_contract_schema"]'
+DISABLE_INFO_LOGS_ENDPOINTS = '["eth_getTransactionById","gen_getContractSchemaForCode","gen_getContractSchema"]'
 
```

Comment:
shouldn't it be **eth_getTransactionByHash** following the recent update?

Reply:
done, thanks!